From c7d982d69081d13cb86160f0e41e355a4d56d320 Mon Sep 17 00:00:00 2001
From: Colin Cross <ccross@android.com>
Date: Mon, 25 Sep 2017 13:40:55 -0700
Subject: [PATCH 15/68] Fix build with proguard

Add extra resource directories to fix proguard errors:
Warning: android.support.design.widget.CoordinatorLayout: can't find referenced class android.support.coreui.R$attr
...
Warning: android.support.v4.media.app.NotificationCompat$DecoratedMediaCustomViewStyle: can't find referenced class android.support.mediacompat.R$layout

Test: m TARGET_PRODUCT=full TARGET_BUILD_VARIANT=userdebug TARGET_BUILD_APPS=Gallery2
Change-Id: Idf556a2eea6af37a4f1f9e9cf337485ae228ff58
---
 Android.mk | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 722988170..8c0a8ef30 100755
--- a/Android.mk
+++ b/Android.mk
@@ -33,10 +33,15 @@ LOCAL_SRC_FILES += $(call all-java-files-under, src_pd)
 # $(warning Anand Commend LOCAL_RENDERSCRIPT_SKIP_INSTALL is $(LOCAL_RENDERSCRIPT_SKIP_INSTALL))
 
 LOCAL_RESOURCE_DIR += $(LOCAL_PATH)/res \
-    $(LOCAL_PATH)/../../../frameworks/support/compat/res
+    $(LOCAL_PATH)/../../../frameworks/support/compat/res \
+    $(LOCAL_PATH)/../../../frameworks/support/media-compat/res \
+    $(LOCAL_PATH)/../../../frameworks/support/core-ui/res
 
 LOCAL_AAPT_FLAGS := --auto-add-overlay
-LOCAL_AAPT_FLAGS += --extra-packages android.support.compat
+LOCAL_AAPT_FLAGS += \
+    --extra-packages android.support.compat \
+    --extra-packages android.support.mediacompat \
+    --extra-packages android.support.coreui \
 
 LOCAL_PACKAGE_NAME := SnapdragonGallery
 
-- 
2.17.1

